client/app/email-domain/general-informations/email-domain-general-informations.controller.js

```diff
@@ -5,36 +5,61 @@ angular.module('App').controller(
      * Constructor
      * @param $scope
      * @param $q
+     * @param $state
```

Comment:
No need for this, you can remove the whole comment

Reply:
Removed